src/app/components/recommendations/md-tabs.scss

```diff
@@ -1,8 +1,20 @@
 md-content {
   background-color: transparent !important;
   md-tabs {
+    min-height: 200px;
     background: #eee;
-    border: 1px solid #ddd;
+    border: 1px solid $primary-color;
+    md-tabs-wrapper {
+      background: white;
+    }
+    & > md-tabs-wrapper > md-tabs-canvas > md-pagination-wrapper > md-ink-bar {
```

Comment:
i cry again, why on earth you keep doing it?
3 inheritance is a max, try to use classes only,
do not do >>>>>, <<< , ~~~~, etc